app/senders/card_mail_sender.rb

```diff
@@ -1,23 +1,29 @@
 class CardMailSender
-  def self.card_moved(card, _)
-    return unless card.changed_list? && card.involved_users.present?
+  def self.card_moved(card, originated_by, move_params)
+    users = card.involved_users - Array(originated_by)
 
-    CardMailer.card_moved(card, card.involved_users).deliver_now
+    return unless card.changed_list? && users.present?
+
+    CardMailer.card_moved(card, users).deliver_now
   end
 
-  def self.card_created(card, _)
-    return unless card.involved_users.present?
+  def self.card_created(card, originated_by, _)
+    users = card.involved_users - Array(originated_by)
+
+    return unless users.present?
 
-    CardMailer.new_card(card, card.involved_users).deliver_now
+    CardMailer.new_card(card, users).deliver_now
   end
 
-  def self.card_archived(card)
-    return unless card.involved_users.present?
+  def self.card_archived(card, originated_by)
+    users = card.involved_users - Array(originated_by)
+
+    return unless users.present?
 
-    CardMailer.card_archived(card, card.involved_users).deliver_now
+    CardMailer.card_archived(card, users).deliver_now
   end
 
-  def self.card_previewed(card, preview_url)
+  def self.card_previewed(card, originated_by, preview_url)
```

Comment:
Unused method argument - `card`. If it's necessary, use `_` or `_card` as an argument name to indicate that it won't be used. You can also write as `card_previewed(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `originated_by`. If it's necessary, use `_` or `_originated_by` as an argument name to indicate that it won't be used. You can also write as `card_previewed(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `preview_url`. If it's necessary, use `_` or `_preview_url` as an argument name to indicate that it won't be used. You can also write as `card_previewed(*)` if you want the method to accept any arguments but don't care about them.